Formula/d/dipc.rb

```diff
@@ -0,0 +1,28 @@
+class Dipc < Formula
+  desc "Convert your favorite images/wallpapers with your favorite color palettes/themes"
+  homepage "https://github.com/doprz/dipc"
+  url "https://github.com/doprz/dipc/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "dd98bf2eea8e97dfaeb8d4e0a991a732e35bf71e1b9bdf0045fdad80e0c0d319"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # Create a simple test PNG file (1x1 pixel, black)
+    (testpath/"test.png").write Base64.decode64(
+      "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==",
+    )
+
+    # Test processing with a built-in theme (gruvbox)
+    output_path = testpath/"output.png"
+    system bin/"dipc", "gruvbox", "--styles", "Dark mode", "-o", output_path, testpath/"test.png"
```

Comment:
```suggestion
    # Test processing with a built-in theme (gruvbox)
    output_path = testpath/"output.png"
    system bin/"dipc", "gruvbox", "--styles", "Dark mode", "-o", output_path, test_fixtures("test.png")
```
I think this should be equivalent.